### ISSUE TITLE
Trouble installing aspire workload using .NET CLI

### ISSUE THREAD

**Issue by christiansparre** [dotnet/aspire#843]
I'm trying to follow the guidance on https://learn.microsoft.com/en-gb/dotnet/aspire/setup-tooling?tabs=dotnet-cli#install-net-aspire

But running `dotnet workload install aspire` just gives me `Workload ID aspire is not recognized.`

I have tried `dotnet workload update` etc. without luck.

I can see on X that I'm not the only one having the same issue (https://twitter.com/Venkatesh_CHN/status/1724643491651969177) and there was a similar "internal?" issue a few weeks back (https://github.com/dotnet/aspire/issues/409)

Any idea what is wrong?

Output of `dotnet --info`

```
.NET SDK:
 Version:           8.0.100
 Commit:            57efcf1350
 Workload version:  8.0.100-manifests.acd1a6d6

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22621
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.100\

.NET workloads installed:
 Workload version: 8.0.100-manifests.acd1a6d6
There are no installed workloads to display.

Host:
  Version:      8.0.0
  Architecture: x64
  Commit:       5535e31a71

.NET SDKs installed:
  6.0.416 [C:\Program Files\dotnet\sdk]
  8.0.100-rc.1.23463.5 [C:\Program Files\dotnet\sdk]
  8.0.100 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0-rc.1.23421.29 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0-rc.1.23419.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.24 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0-rc.1.23420.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]
```

Was really looking forward to trying this as we use `tye` currently and that has been abandoned.

**Comment by DamianEdwards** [comment]
Can you try running `dotnet workload update` and then running `dotnet workload install aspire` after that?

**Comment by DamianEdwards** [comment]
Can you add the output of `dotnet nuget list source` please?

**Comment by christiansparre** [comment]
Yes, it is:

```
  1.  nuget.org [Enabled]
      https://api.nuget.org/v3/index.json
  2.  dotnet8 [Enabled]
      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json
  3.  Microsoft Visual Studio Offline Packages [Enabled]
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
```

The dotnet8 is there because I tried a lot of different things to try getting it to work

I removed that source so it is now:

```
  1.  nuget.org [Enabled]
      https://api.nuget.org/v3/index.json
  2.  Microsoft Visual Studio Offline Packages [Enabled]
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
```

and I still get `Workload ID aspire is not recognized.`

I notice that the guidance is mention `Visual Studio 2022 version 17.9 or higher (Optional)`, I just went for the CLI route. immediately.

I only have 17.8 installed

**Comment by joperezr** [comment]
Interesting, I would have expected this to work. Can you please confirm that you have a folder called `microsoft.net.sdk.aspire` under `C:\Program Files\dotnet\sdk-manifests\8.0.100\`, and if so, can you confirm what version subfolders are under that aspire folder?

Also, can you try typing `dotnet workload search` and share the output to see if aspire is listed there?

**Comment by DamianEdwards** [comment]
OK we think we understand the issue here. It's due to how the 8.0 SDK is authored into VS 17.8

Two potential workarounds for now:

- Try installing the .NET 8 SDK manually and then installing the aspire workload
- Install VS 17.9 Preview 1 (it's side by side with 17.8)

**Comment by christiansparre** [comment]
Manually installed the .NET 8 SDK and `dotnet workload install aspire` immediately worked 🎉💪 

Thank you for the quick responses, you rock!

**Comment by riverar** [comment]
@timheuer Should we get a PR started for edits to https://learn.microsoft.com/dotnet/aspire/setup-tooling to let folks know this is known broken (in .NET 8 SDK delivered via Visual Studio 17.8)?

**Comment by timheuer** [comment]
> Is it only appear for VS 2022 Preview?

Yes the VS support is only available in 17.9 Preview

**Comment by riverar** [comment]
@IEvangelist I don't think the edits are sufficient here. The Aspire workload is present in both retail and preview .NET SDKs. It's just that if users go down the route of using the .NET SDK bundled with Visual Studio, the Aspire workload is missing.

**Comment by niemyjski** [comment]
I'm running into this on one of my machines running latest (VS2022/Windows11) non preview bits. The cli search is not finding any workloads even being on the latest .NET 8 SDK and returns immediately.

```
PS D:\Code> dotnet workload update --from-previous-sdk

No workloads installed for this feature band. To update workloads installed with earlier SDK versions, include the --from-previous-sdk option.
Updated advertising manifest microsoft.net.sdk.macos.
Updated advertising manifest microsoft.net.workload.mono.toolchain.net6.
Updated advertising manifest microsoft.net.sdk.android.
Updated advertising manifest microsoft.net.sdk.maccatalyst.
Updated advertising manifest microsoft.net.workload.mono.toolchain.current.
Updated advertising manifest microsoft.net.sdk.maui.
Updated advertising manifest microsoft.net.workload.emscripten.net7.
Updated advertising manifest microsoft.net.sdk.tvos.
Updated advertising manifest microsoft.net.workload.emscripten.current.
Updated advertising manifest microsoft.net.workload.emscripten.net6.
Updated advertising manifest microsoft.net.sdk.ios.
Updated advertising manifest microsoft.net.workload.mono.toolchain.net7.
No workloads installed for this feature band. To update workloads installed with earlier SDK versions, include the --from-previous-sdk option.
The machine has a pending reboot. The workload operation will continue, but you may need to restart.

Successfully updated workload(s): .

PS D:\Code> dotnet workload install aspire
Workload ID aspire is not recognized.
PS D:\Code> dotnet --list-sdks
6.0.202 [C:\Program Files\dotnet\sdk]
8.0.101 [C:\Program Files\dotnet\sdk]
```

**Comment by niemyjski** [comment]
After upgrading to the preview it seems to confirm it was picking up existing workloads (form 6.0 sdk?). I was presented with this:
```
PS D:\Code> dotnet workload update --from-previous-sdk

Welcome to .NET 8.0!
---------------------
SDK Version: 8.0.200-preview.23624.5
```

and it worked as expected.

**Comment by IEvangelist** [comment]
> @IEvangelist I don't think the edits are sufficient here. The Aspire workload is present in both retail and preview .NET SDKs. It's just that if users go down the route of using the .NET SDK bundled with Visual Studio, the Aspire workload is missing.

I'm pretty sure that the .NET Aspire tooling docs are solid now, we have two paths, Visual Studio or .NET CLI, but include validation steps and the one for VS shows the user how to correctly select the workload. Is there anything that we're missing?

**Comment by tadams79** [comment]
@IEvangelist ,

Actually, I'm not sure that they are. I have the latest builds of .NET 8.0 and Visual Studio Preview 17.9.1. I cannot get any workloads to install. I keep receiving error messages that the workloads are unsupported on my platform. I'm running Windows 11 Professional, Visual Studio 2022 Preview 17.9.1 (Pro). I've attempted to remove the bundled .NET 8 and install from the .NET link and no joy. Below are the outputs:

### dotnet info

```
.NET SDK:
 Version:           8.0.100
 Commit:            57efcf1350
 Workload version:  8.0.100-manifests.ba313bcd

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22631
 OS Platform: Windows
 RID:         win-x86
 Base Path:   C:\Program Files (x86)\dotnet\sdk\8.0.100\

.NET workloads installed:
 Workload version: 8.0.100-manifests.ba313bcd
There are no installed workloads to display.

Host:
  Version:      8.0.0
  Architecture: x86
  Commit:       5535e31a71

.NET SDKs installed:
  8.0.100 [C:\Program Files (x86)\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 7.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.0 [C:\Program Files (x86)\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.10 [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.15 [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.0 [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 5.0.10 [C:\Program Files (x86)\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files (x86)\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x64   [C:\Program Files\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x64\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found
```

### Nuget Sources
```
Registered Sources:
  1.  nuget.org [Enabled]
      https://api.nuget.org/v3/index.json
  2.  Infragistics (Local) [Enabled]
      C:\Users\Public\Documents\Infragistics\NuGet\
  3.  DevExpress 23.2 Local [Enabled]
      C:\Program Files\DevExpress 23.2\Components\System\Components\Packages
```
### Workload Search
```
Workload ID       Description
--------------------------------------------------------------------
macos             .NET SDK Workload for building macOS applications.
maui-tizen        .NET MAUI SDK for Tizen
maui-windows      .NET MAUI SDK for Windows
```
### Workload Update
```
PS C:\Users\Administrator> dotnet workload update --from-previous-sdk

No workloads installed for this feature band. To update workloads installed with earlier SDK versions, include the --from-previous-sdk option.
Updated advertising manifest microsoft.net.sdk.ios.
Updated advertising manifest microsoft.net.workload.emscripten.current.
Updated advertising manifest microsoft.net.sdk.tvos.
Updated advertising manifest microsoft.net.workload.mono.toolchain.net6.
Updated advertising manifest microsoft.net.workload.emscripten.net6.
Updated advertising manifest microsoft.net.workload.emscripten.net7.
Updated advertising manifest microsoft.net.sdk.maccatalyst.
Updated advertising manifest microsoft.net.workload.mono.toolchain.current.
Updated advertising manifest microsoft.net.sdk.aspire.
Updated advertising manifest microsoft.net.sdk.maui.
Updated advertising manifest microsoft.net.sdk.macos.
Updated advertising manifest microsoft.net.workload.mono.toolchain.net7.
Updated advertising manifest microsoft.net.sdk.android.
No workloads installed for this feature band. To update workloads installed with earlier SDK versions, include the --from-previous-sdk option.

Successfully updated workload(s):
```

### Error Message
```
PS C:\Users\Administrator> dotnet workload install aspire
Workload ID aspire isn't supported on this platform.
```

**Comment by DamianEdwards** [comment]
@tadams79 have you downloaded and installed the .NET 8.0.101 SDK from here and installed that? Also please check you have the latest VS updates installed.

I just noticed you appear to be using the x86 .NET SDK. Is that intentional? Can you tell us more about your machine?

**Comment by tadams79** [comment]
@DamianEdwards ,

My machine is running a Ryzen 5 5000 series CPU, 16GB of RAM, 500 GB NVMe drive as main and a 500 Intel SSD for storage of projects. I didn't realize I was using the x86 .NET SDK. That is not intentional. I'll also download the .NET 8.0.101 SDK and install the x64 version.

**Comment by tadams79** [comment]
I changed to a different computer, downloaded and installed the latest .NET 8.0.101 SDK, ran the dotnet --info command and the output was as expected. I then ran dotnet workload search and my results are as noted below. I think I am going to format the other hard drive and start over. There's clearly something a miss there. Thanks for the help.

```
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------

Workload ID                 Description
------------------------------------------------------------------------------------
android                     .NET SDK Workload for building Android applications.
aspire                      .NET Aspire SDK (Preview)
ios                         .NET SDK Workload for building iOS applications.
maccatalyst                 .NET SDK Workload for building MacCatalyst applications.
macos                       .NET SDK Workload for building macOS applications.
maui                        .NET MAUI SDK for all platforms
maui-android                .NET MAUI SDK for Android
maui-desktop                .NET MAUI SDK for Desktop
maui-ios                    .NET MAUI SDK for iOS
maui-maccatalyst            .NET MAUI SDK for Mac Catalyst
maui-mobile                 .NET MAUI SDK for Mobile
maui-tizen                  .NET MAUI SDK for Tizen
maui-windows                .NET MAUI SDK for Windows
mobile-librarybuilder       workloads/mobile-librarybuilder/description
tvos                        .NET SDK Workload for building tvOS applications.
wasi-experimental           workloads/wasi-experimental/description
wasm-experimental           workloads/wasm-experimental/description
wasm-experimental-net7      .NET WebAssembly experimental tooling for net7.0
wasm-tools                  .NET WebAssembly build tools
wasm-tools-net6             .NET WebAssembly build tools for net6.0
wasm-tools-net7             .NET WebAssembly build tools for net7.0
```

**Comment by tadams79** [comment]
For others who are having a similar issue, remember that there's two dotnet paths. I went into Environment Variables, and edited "Path". I removed the x86 from the path and all is working just fine now.

**Comment by davidfowl** [comment]
@joperezr There's nothing more to fix here right?

**Comment by riverar** [comment]
@davidfowl This is still not working.

Steps to reproduce:
1. Clean machine
2. Install VS Community 2022 to get .NET 8.0.101 SDK pulled down
3. `dotnet workload update`
4. `dotnet workload install aspire`

**Comment by timheuer** [comment]
@riverar can you confirm two things: VS version should be Preview (17.9+) and in that installation you had just enabled 'ASP.NET Workload' and didn't change defaults?

**Comment by riverar** [comment]
I'm using retail Visual Studio to demonstrate the confusion resulting from the docs showing a success path using .NET SDK 8.0, which VS retail pulls down.

So the docs need to be explicit and denounce the .NET SDK 8.0 that ships with VS retail.

**Comment by riverar** [comment]
That doc page doesn't have contributions enabled, but here's a suggestion to quickly close this issue out:

```
To work with .NET Aspire, you'll need the following installed locally:
  * .NET 8.0
  * .NET Aspire workload
  * Docker Desktop
  * Integrated Developer Environment (IDE) or code editor, such as:
    - Visual Studio 2022 Preview version 17.9 or higher (Optional)
    - Visual Studio Code (Optional)

NOTE: The .NET Aspire workload is not available via the .NET 8.0 SDK installed by Visual Studio 2022.
To access this workload on machines without Visual Studio 2022 Preview installed, install the standalone
.NET 8.0 SDK (link).
```

**Comment by timheuer** [comment]
Got it thanks for clarifying intent.  We're at an inbetween state for sure as what I just did on clean machine:
- Installed latest VS preview (with ASPNET workload option -- which by default includes .NET Aspire)
- post install state: .NET 8.0.200 SDK preview + .NET Aspire preview 1
- `dotnet workload update`
- end state .NET Aspire preview 2 installed via update

@davidfowl I think next standup we need to talk through this with @joperezr / @marcpopMSFT (and @IEvangelist) to figure out the varied states we may be in now until preview 3.

**Comment by IEvangelist** [comment]
Hi @riverar,

> That doc page doesn't have contributions enabled, but here's a suggestion to quickly close this issue out:

We accept contributions, and they are enabled. Lots of folks have been providing contributions... Select the pencil icon on the document in question, and it will open the corresponding markdown file used to generate the HTML:

![image](https://github.com/dotnet/aspire/assets/7679720/39c7e924-6c2c-401b-9d5c-4b73251bf0de)

From there, it's easy to make a change.

**Comment by marcpopMSFT** [comment]
@timheuer customers seem to be trying aspire with 17.8 a lot in this discussion. We could add the aspire manifest to the 17.8 authoring which would allow them to install the workload but I don't know if they'd get blocked further down the workflow with missing VS pieces. I wonder if we should consider a custom error if you try to install aspire when the manifest isn't present. That's a bit hacky hardcoding a specific workload.

At the very least, there should be a release note that covers the workload is not recognized error.

Most people in this thread appear to be trying aspire with 17.8 and that's the primary issue. One person hit https://aka.ms/dotnet/x86path as well.

**Comment by riverar** [comment]
But to be clear, the aspire workload is available in the .NET SDK that ships with 17.8. This isn't an issue where users want to use the wrong VS.  It's an issue with VS delivering a broken SDK.

**Comment by joperezr** [comment]
FWIW, I believe the issue with VS 17.8 where the SDK doesn't have the Aspire manifest in it was fixed so servicing versions of 17.8 should not have an issue any longer. @joeloff @danegsta is that right?

**Comment by MithrilMan** [comment]
@joperezr 
In my env I've just update VS2022 to 17.8.6 and it's not working

```
C:\Users\xyz>dotnet --info
.NET SDK:
 Version:           8.0.101
 Commit:            6eceda187b
 Workload version:  8.0.100-manifests.1c08acc9

Ambiente di runtime:
 OS Name:     Windows
 OS Version:  10.0.22631
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.101\

Carichi di lavoro .NET installati:
 Workload version: 8.0.100-manifests.1c08acc9
 [android]
   Origine dell'installazione: VS 17.8.34525.116
   Versioni del manifesto:    34.0.52/8.0.100
   Percorso del manifesto:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.android\34.0.52\WorkloadManifest.json
   Tipo di installazione:              Msi

 [maccatalyst]
   Origine dell'installazione: VS 17.8.34525.116
   Versioni del manifesto:    17.2.8004/8.0.100
   Percorso del manifesto:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maccatalyst\17.2.8004\WorkloadManifest.json
   Tipo di installazione:              Msi

 [wasm-tools]
   Origine dell'installazione: VS 17.8.34525.116
   Versioni del manifesto:    8.0.1/8.0.100
   Percorso del manifesto:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.workload.mono.toolchain.current\8.0.1\WorkloadManifest.json
   Tipo di installazione:              Msi

 [maui-windows]
   Origine dell'installazione: VS 17.8.34525.116
   Versioni del manifesto:    8.0.3/8.0.100
   Percorso del manifesto:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.maui\8.0.3\WorkloadManifest.json
   Tipo di installazione:              Msi

 [ios]
   Origine dell'installazione: VS 17.8.34525.116
   Versioni del manifesto:    17.2.8004/8.0.100
   Percorso del manifesto:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.ios\17.2.8004\WorkloadManifest.json
   Tipo di installazione:              Msi


Host:
  Version:      8.0.1
  Architecture: x64
  Commit:       bf5e279d92

.NET SDKs installed:
  6.0.418 [C:\Program Files\dotnet\sdk]
  8.0.101 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.26 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.26 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.26 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 8.0.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
```

**Comment by balachir** [comment]
I believe this issue is now fixed in the latest versions of VS 17.9.0 and higher. Hence, I think we can close this issue now.

@joperezr @timheuer any concerns with closing this?

**Comment by joperezr** [comment]
Closing sounds good to me, as the problem is identified and fixed in later versions of VS, as well as for folks hitting this that can't upgrade VS we have a clear workaround (manually installing the 8.0 SDK)

**Comment by riverar** [comment]
I tested this and can confirm it's working in 17.9+ that is no longer in preview.

(We already knew this was fixed in 17.9, we were just trying to get the docs and experience iron out. But it took so long that 17.9 caught up.)

**Comment by balachir** [comment]
@riverar we apologize for the trouble caused and we'll work to improve this in the future. We ended up in this situation due to unfortunate timing of the releases and the December holiday period. As you correctly pointed out in one of your earlier posts, the issue was originally with the .NET SDK 8.0.100 that shipped in VS 17.8.0. By the time we investigated this and the fix was understood, the builds of .NET SDK 8.0.101 and VS 17.8.1 had already been locked down in December for the Jan update. Our next opportunity to fix this was for the Feb update - SDK 8.0.200 and VS 17.9.0 and this is the release when the issue got fixed.
